release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -148,6 +148,90 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
+
+For more information, see xref:../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc[CSI inline ephemeral volumes].
+
+[id="ocp-4-13-storage-csi-migration-azure-file-GA"]
+==== Automatic CSI migration for Microsoft Azure File auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Azure File was provided in this feature in {product-title} 4.10. {product-title} 4.13 now supports automatic migration for Azure File as generally available. CSI migration for Azure File is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+Although storage class referencing to the in-tree storage plugin will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+[id="ocp-4-13-storage-csi-migration-vsphere-GA"]
+==== Automatic CSI migration for VMware vSphere auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for vSphere was provided in this feature in {product-title} 4.10. {product-title} 4.13 now supports automatic migration for vSphere as generally available. CSI migration for vSphere is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+Although storage class referencing to the in-tree storage plugin will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+[id="ocp-4-13-storage-aws-efs-cross-account-support"]
+==== Cross account support for AWS EFS CSI driver is generally available
+Cross account support allows you to have an {product-title} cluster in one Amazon Web Services (AWS) account and mount your file system in another AWS account using the AWS Elastic File System (EFS) Container Storage Interface (CSI) driver.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-fsgroup-to-csi-driver"]
+==== Delegate FSGroup to CSI Driver instead of Kubelet (Technology Preview)
```

Comment:
https://issues.redhat.com/browse/STOR-843 I think it is a GA feature.
@fabiand Can you help to double confirm?

Reply:
Yes this is GA